zerver/tests/test_openapi.py

```diff
@@ -318,7 +318,8 @@ def get_type_by_priority(
         val = 6
         for t in types:
             if isinstance(t, tuple):
-                return t  # e.g. (list, dict) or (list, str)
+                # e.g. (list, dict) or (list, str)
+                return t  # nocoverage
```

Comment:
I assume this loses coverage for the same reason? Probably OK, I assume we may end up deleting a bunch of this code once the Pydantic transition is complete.